src/components/ComponentRelation/cr-modals.tsx

```diff
@@ -84,8 +84,14 @@ export const DefineComponentRelationModal: React.FC<DefineComponentRelationModal
                         sortedGroupedComponents={sortedGroupedComponents}
                         index={index}
                         removeProps={{
-                          disableRemove: values.relations.length <= 1,
-                          onRemove: () => arrayHelpers.remove(index),
+                          onRemove: () =>
+                            values.relations.length <= 1
+                              ? arrayHelpers.replace(0, {
```

Comment:
hmm, we also need to allow the user to save changes when the user remove the only one relantioship. Otherwise it would give a false impression that the relationship was remove, but once they open the modal again, the relationship would still be there. Example:

https://github.com/user-attachments/assets/ee1d6754-91d0-4bfe-a7ec-ccf9585af88d